README.md

```diff
@@ -278,6 +278,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [OCR.Space](https://ocr.space/) — An OCR API which parses image and pdf files returning the text results in JSON format. 25,000 requests per month free.
   * [Duply.co](https://duply.co) — Create dynamic images from API & URL, design template once and reuse it. Free tier offers 70 images/month creation from API & URL and Up to 100 through Form.
   * [Frontend Zero to One](https://www.fezto.xyz/) — Launch an app immediately from your OpenAPI or Swagger spec, see how partners experience your API. Free. Paid tier for customizations and code.
+  * [Open AI](https://beta.openai.com/docs/introduction/overview) — Can be used for tasks such as language translation, summarization, and question answering.
```

Comment:
<img width="563" alt="Screenshot 2023-01-09 at 08 44 34" src="https://user-images.githubusercontent.com/82342/211260613-43bf3569-876e-442b-9cd6-b156cbd0bce0.png">

Does not seem to be a free tier at all.